packages/web/src/components/ui/CardList/CardList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './CardList.module.scss';
+import Card, { CardProps } from '../Card/Card';
+import clsx from 'clsx';
+
+export type CardListProps = {
+  cards: CardProps[];
+  variant?: 'default' | 'vertical' | string;
+};
+
+export default function CardList({ cards, variant = 'default' }: CardListProps) {
+  const style = { '--total': cards.length } as React.CSSProperties;
```

Comment:
I'm not sure about css variables inside `style` attribute. @zardak @ctizen what do you think?

Reply:
In order for this horizontal scrollable grid container on mobile to work it needs an external total items counter coming from JS, so it can properly calculate `grid-template-columns`. [Working example and reference](https://codepen.io/dannievinther/pen/PdvvQv)

[CSS-only example](https://codepen.io/dannievinther/pen/PdvvQv) leaves huge gap at the end of container:
![image](https://user-images.githubusercontent.com/2999208/186876035-2a95674a-8525-4f9f-bd97-ddc8cac421ac.png)

Reply:
Is grid even required here? Looks like it can be implemented with a plain container with fixed-width inline blocks within.

Reply:
Yep. Flexbox solution would be sufficient enough.

Reply:
> I'm not sure about css variables inside style attribute.

I'd say this is not a bad practice per-se, if used sparingly. However, we should try solving this specific case with flexbox, unless we need to snap the scroll to items.